src/gt4py/next/embedded/context.py

```diff
@@ -62,4 +60,4 @@ def ctx_updater(*args: tuple[cvars.ContextVar[Any], Any]) -> None:
 
 
 def within_context() -> bool:
```

Comment:
I'd rename it to something like `within_valid_context()`  and add a docstring to this function also.